modules/ztp-querying-the-policy-compliance-status-for-each-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-querying-the-policy-compliance-status-for-each-cluster_{context}"]
+= Querying the policy compliance status for each cluster
+
+After you have created the validator inform policies for your clusters and pushed them to the
+the zero touch provisioning (ZTP) Git repository, you can check the status of each cluster for policy compliance.
+
+.Procedure
+
+. To query the status of the spoke clusters using either the {rh-rhacm-first} web console or the CLI:
++
+* To query status from the {rh-rhacm} web console, click *Governance => Find policies =>
+Search “du-validator-policy” => Click into the policy*.
+
+* To query status using the CLI, run the following command:
++
+[source,terminal]
+----
+$ oc get policies du-validator-policy -n <namespace-for-common> -o jsonpath={'.status.status'} | jq
+----
++
+When all of the policies including the validator inform policy applied to the cluster become compliant,
+it means the ZTP installation and configuration for this cluster is complete.
+
+. To query the cluster violation/compliant status from the ACM web console, click
+*Governance => Cluster violations*.
+
+. Check the validator policy compliant status for a cluster using the following commands:
++
+[source,terminal]
+----
+$ export CLUSTER=<clusterName>
+----
++
+[source,terminal]
+----
+$ oc get policies -n $CLUSTER | grep <validatorPolicyName>
+----
++
+You can also use the following command:
++
+[source,terminal]
+----
+$ oc get policies -n <namespace-for-group> <validatorPolicyName> -o jsonpath="{.status.status[?(@.clustername=='$CLUSTER')]}" | jq
+----
++
+After the ``*-validator-du-policy` {rh-rhacm} policy becomes compliant for the cluster, the
```

Comment:
Looks like an extra backtick here. I don't think the `*` needs to be escaped or anything tho, this seems to work fine in Atom at least:

```
After the `*-validator-du-policy` {rh-rhacm} policy
```

Reply:
This sentence overall is a bit long too, consider maybe breaking it up.

Reply:
changed